src/docs/stan-reference/functions.tex

```diff
@@ -1818,6 +1817,29 @@ \section{Array Broadcasting}\label{array-broadcasting.section}
 \code{k} in \code{1:4}, \code{m} in \code{1:5}, and \code{n} in
 \code{1:6}.
 
+%
+\section{Array Concatenation}\label{array-concatenation.section}
+
+\begin{description}
+\fitem{T}{append\_array}{T \farg{x}, T \farg{y}}{
+  Return the concatenation of two arrays in the order of the arguments.
+  T must be an N-dimensional array of any Stan type (with a maximum N of 8). All
```

Comment:
I think this is actually a maximum N of 7, as the loop goes from 1 through `N < 8`.